llvm/docs/GettingStarted.rst

```diff
@@ -286,15 +286,15 @@ uses the package and provides other details.
 Package                                                     Version      Notes
 =========================================================== ============ ==========================================
 `CMake <http://cmake.org/>`__                               >=3.20.0     Makefile/workspace generator
-`python <http://www.python.org/>`_                          >=3.6        Automated test suite\ :sup:`1`
+`python <http://www.python.org/>`_                          >=3.8        Automated test suite\ :sup:`1`
 `zlib <http://zlib.net>`_                                   >=1.2.3.4    Compression library\ :sup:`2`
 `GNU Make <http://savannah.gnu.org/projects/make>`_         3.79, 3.79.1 Makefile/build processor\ :sup:`3`
 =========================================================== ============ ==========================================
 
 .. note::
 
    #. Only needed if you want to run the automated test suite in the
-      ``llvm/test`` directory.
+      ``llvm/test`` directory or the libclang Python binding unit tests.
```

Comment:
I would change up the wording here. There are many other facilities within LLVM that also need Python. I'd specifically mention the test suite and then wrap everything else into a generic term.

Reply:
Checkout the latest commits. I gave it a shot, but I might not have enough context of other LLVM Python projects to accurately categorize them.

Reply:
Please reopen if you think this can be further improved.

Reply:
LGTM. I think that should encompass the uses in tree, even including subprojects like LLDB, although I guess those bindings aren't fully auxiliary as they're on by default. The LLDB build documentation is separate though.

Reply:
True. I'm more than happy to change the sentence to correctly represent the state of the world.

Feel free to suggest or maybe the following?

```
Only needed if you want to run the automated test suite in the
``llvm/test`` directory, or if you plan to utilize any Python libraries or utilities.
```

or "LLVM Python libraries or utilities."

Reply:
I'd mention bindings in their too and then I think that should pretty much cover everything:
```
...or if you plan to utilize any Python libraries, utilities, or bindings.
```

Reply:
Done.